hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/Message.java

```diff
@@ -81,18 +88,22 @@ public boolean equals(Object obj) {
             return false;
         }
         final Message other = (Message) obj;
-        return Objects.equals(this.offset, other.offset)
-                && Objects.equals(this.partition, other.partition)
-                && Objects.equals(this.topic, other.topic)
+        return Objects.equals(this.topic, other.topic)
```

Comment:
do we need to check all these fields? from *business* point of view, i guess partition, offset and topic (content type is in topic name?) is uuid